### PR TITLE
Fixed icon fill colors

### PR DESCRIPTION
**Pull request by shubham1206agra**
### Fixed Issues

$ https://github.com/Expensify/App/issues/33664

### Tests

1. Go to Workspace > Bank Account
2. Select Connect online via plaid
3. Select any bank, and complete the plaid process
4. Observe the icon next to bank name

- [x] Verify that no errors appear in the JS console

### Offline tests

Same as Tests

### QA Steps

Same as Tests

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

Screenshot_2023-12-27-08-04-04-837_com expensify chat adhoc

Screenshot_2023-12-27-08-04-59-680_com expensify chat adhoc

</details>

<details>
<summary>Android: mWeb Chrome</summary>

</details>

<details>
<summary>iOS: Native</summary>

Screenshot 2023-12-27 at 7 18 42 PM

</details>

<details>
<summary>iOS: mWeb Safari</summary>

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

Screenshot 2023-12-27 at 7 33 43 PM

</details>

<details>
<summary>MacOS: Desktop</summary>

Screenshot 2023-12-26 at 11 00 53 PM

</details>

### CODE REVIEW

**Comment by shubham1206agra** [comment]
Waiting for https://expensify.slack.com/archives/C049HHMV9SM/p1703677023896169 to get an issue

**Comment by shubham1206agra** [comment]
@Pujan92 Would you like to start the review in the meantime?

**Comment by Pujan92** [comment]
> @Pujan92 Would you like to start the review in the meantime?

Yes.

File: src/components/AddPlaidBankAccount.js [hunk 1]
```diff
@@ -249,7 +249,7 @@ function AddPlaidBankAccount({
                     height={iconSize}
                     width={iconSize}
                     additionalStyles={iconStyles}
-                    fill={theme.icon}
+                    displayInDefaultIconColor
```
Comment (Pujan92) [inline src/components/AddPlaidBankAccount.js:252]: We don't require to pass displayInDefaultIconColor here as it is only used inside MenuItem.

**Comment by Pujan92** [comment]
We also need to remove the fill prop within `ImageCropView` component.
https://github.com/Expensify/App/pull/30905/files#diff-7ead4ae598b36d2acf038cfe3d0a166b21e9a9ebe6da3d5e72fe7e5d6877d8caR93

File: src/components/AddPlaidBankAccount.js (hunk 1 above)
Comment (shubham1206agra) [reply src/components/AddPlaidBankAccount.js:252]: MenuItem has default fill color if `displayInDefaultIconColor` is not specified

Comment (Pujan92) [reply src/components/AddPlaidBankAccount.js:252]: But this is simply an Icon not a MenuItem.

**Comment by Pujan92** [comment]
> > We also need to remove the fill prop within `ImageCropView` component. https://github.com/Expensify/App/pull/30905/files#diff-7ead4ae598b36d2acf038cfe3d0a166b21e9a9ebe6da3d5e72fe7e5d6877d8caR93
> 
> This is not entirely possible as it uses 2 colors. We can use `react-native-svg` to generate the svg dynamically for theme.

Sorry but I am not getting it, do you mean it is added intentionally to differentiate mask image for diff theme?
If yes then let me share the screenshot of the comparison.

**Comment by shubham1206agra** [comment]
Nope, I am talking about creating a custom svg that switches colors on switching theme. And using that instead. See Lounge svg for reference

**Comment by shubham1206agra** [comment]
@mountiny What's your opinion on https://github.com/Expensify/App/pull/33607#issuecomment-1874054095?

File: src/components/AddPlaidBankAccount.js (hunk 1 above)
Comment (Pujan92) [reply src/components/AddPlaidBankAccount.js:252]: @shubham1206agra  Do you agree that `displayInDefaultIconColor` is irrelevant here?

Comment (shubham1206agra) [reply src/components/AddPlaidBankAccount.js:252]: Oh right. Overlooked it.

**Comment by dannymcclain** [comment]
In reference to [these screenshots](https://github.com/Expensify/App/pull/33607#issuecomment-1874054095), I think I have a slight preference for the left side (without fill prop), but honestly both look acceptable. @Expensify/design care to weigh in here?

**Comment by shawnborton** [comment]
What exactly is the question? I'm a bit lost. 

Are we discussing how to handle the default bank account icon in light mode vs dark mode? If so, from a technical perspective, I don't feel too strongly in terms of how we implement it, but I do agree that each theme should use its respective colors for these default icons.

**Comment by Pujan92** [comment]
The question is with the latest changes we landed on the right side screenshots, but seems it applied by mistake in that PR. So we were thinking of reverting that which makes it look like the left side(the way it was for long), but as we are not sure we wanted to clarify which side you prefer.

**Comment by mountiny** [comment]
@shawnborton the question is about this comment and screenshots in it specifically https://github.com/Expensify/App/pull/33607#issuecomment-1874054095

**Comment by shawnborton** [comment]
Ah got it. I would think that the slightly translucent color of that mask should match the app BG in each respective theme. So I think I like these two: 

Dark | Light 
----- | -----
image | image


And then perhaps we should adjust border color to use the theme border color too?

**Comment by dannymcclain** [comment]
@shawnborton that sounds good to me!

**Comment by Pujan92** [comment]
@shubham1206agra I believe that(mask image based on theme) should be handled in a separate issue as for this we are just correcting the prop set in the wrong places. Do you think the same?

**Comment by Pujan92** [comment]
> @shubham1206agra @Pujan92 Can we just do it here? I think its not big enough that it could not be added to this PR?

Yes, @shubham1206agra I think for maskimage below changes should work.
1) Pass `theme.iconReversed` for the fill prop instead of `theme.icon`.
https://github.com/Expensify/App/pull/30905/files#diff-7ead4ae598b36d2acf038cfe3d0a166b21e9a9ebe6da3d5e72fe7e5d6877d8caR93
2) Remove fixed fill from the class `st1` in image-crop-circle-mask.svg

**Comment by shubham1206agra** [comment]
@allroundexperts Here @Pujan92 will do the review. Sorry for the tag.

**Comment by mountiny** [comment]
@Pujan92 let us know your eta, thanks!

**Comment by Pujan92** [comment]
## Reviewer Checklist
* [x]  I have verified the author checklist is complete (all boxes are checked off).
* [x]  I verified the correct issue is linked in the `### Fixed Issues` section above
* [x]  I verified testing steps are clear and they cover the changes made in this PR
   
   * [x]  I verified the steps for local testing are in the `Tests` section
   * [x]  I verified the steps for Staging and/or Production testing are in the `QA steps` section
   * [x]  I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
   * [x]  I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
 * [x]  I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
 * [x]  I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
 * [x]  I verified tests pass on **all platforms** & I tested again on:
   
   * [x]  Android: Native
   * [x]  Android: mWeb Chrome
   * [x]  iOS: Native
   * [x]  iOS: mWeb Safari
   * [x]  MacOS: Chrome / Safari
   * [x]  MacOS: Desktop
 * [x]  If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
 * [x]  I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
   
   * [x]  I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
   * [x]  I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
   * [x]  I verified that comments were added to code that is not self explanatory
   * [x]  I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
   * [x]  I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
   * [x]  I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
   * [x]  I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
   * [x]  I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
   * [x]  I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
 * [x]  If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
 * [x]  I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
 * [x]  I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
 * [x]  I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
 * [x]  I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
 * [x]  If a new component is created I verified that:
   
   * [x]  A similar component doesn't exist in the codebase
   * [x]  All props are defined accurately and each prop has a `/** comment above it */`
   * [x]  The file is named correctly
   * [x]  The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
   * [x]  The only data being stored in the state is data necessary for rendering and nothing else
   * [x]  For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
   * [x]  Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
   * [x]  All JSX used for rendering exists in the render method
   * [x]  The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
 * [x]  If any new file was added I verified that:
   
   * [x]  The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
 * [x]  If a new CSS style is added I verified that:
   
   * [x]  A similar style doesn't already exist
   * [x]  The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
 * [x]  If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
 * [x]  If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
 * [x]  If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
 * [x]  If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
 * [x]  If the PR modifies the form input styles:
   
   * [x]  I verified that all the inputs inside a form are aligned with each other.
   * [x]  I added `Design` label so the design team can review the changes.
 * [x]  If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
 * [x]  If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
 * [x]  I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.
 
 ### Screenshots/Videos
<details>
<summary>Web</summary>

Screenshot 2024-01-08 at 09 18 50

</details>

<details>
<summary>Mobile Web - Chrome</summary>


</details>

<details>
<summary>Mobile Web - Safari</summary>


</details>

<details>
<summary>Desktop</summary>


</details>

<details>
<summary>iOS</summary>

Simulator Screen Shot - iPhone 14 Pro Max - 2024-01-08 at 09 01 23
Simulator Screen Shot - iPhone 14 Pro Max - 2024-01-08 at 09 01 54
Simulator Screen Shot - iPhone 14 Pro Max - 2024-01-08 at 09 16 48

</details>

<details>
<summary>Android</summary>



</details>

**Review by Pujan92 — APPROVED** [review]
Looks good to me. Seems need to re-run the flow to consider a new reviewer checklist.
cc: @mountiny

**Comment by Pujan92** [comment]
## Reviewer Checklist

- [X] I have verified the author checklist is complete (all boxes are checked off).
- [X] I verified the correct issue is linked in the `### Fixed Issues` section above
- [X] I verified testing steps are clear and they cover the changes made in this PR
    - [X] I verified the steps for local testing are in the `Tests` section
    - [X] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [X] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [X] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [X] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [X] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [X] I verified tests pass on **all platforms** & I tested again on:
    - [X] Android: Native
    - [X] Android: mWeb Chrome
    - [X] iOS: Native
    - [X] iOS: mWeb Safari
    - [X] MacOS: Chrome / Safari
    - [X] MacOS: Desktop
- [X] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [X] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [X] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [X] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [X] I verified that comments were added to code that is not self explanatory
    - [X] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [X] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [X] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [X] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [X] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [X] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [X] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [X] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [X] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [X] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [X] If a new component is created I verified that:
    - [X] A similar component doesn't exist in the codebase
    - [X] All props are defined accurately and each prop has a `/** comment above it */`
    - [X] The file is named correctly
    - [X] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [X] The only data being stored in the state is data necessary for rendering and nothing else
    - [X] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [X] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [X] All JSX used for rendering exists in the render method
    - [X] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [X] If any new file was added I verified that:
    - [X] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [X] If a new CSS style is added I verified that:
    - [X] A similar style doesn't already exist
    - [X] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [X] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [X] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [X] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [X] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [X] If the PR modifies the form input styles:
    - [X] I verified that all the inputs inside a form are aligned with each other.
    - [X] I added `Design` label so the design team can review the changes.
- [X] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [X] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [X] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

 ### Screenshots/Videos
<details>
<summary>Web</summary>

Screenshot 2024-01-08 at 09 18 50

</details>

<details>
<summary>Mobile Web - Chrome</summary>


</details>

<details>
<summary>Mobile Web - Safari</summary>

Simulator Screen Shot - iPhone 14 Pro Max - 2024-01-08 at 09 39 08

</details>

<details>
<summary>Desktop</summary>

Screenshot 2024-01-08 at 09 45 18


</details>

<details>
<summary>iOS</summary>

Simulator Screen Shot - iPhone 14 Pro Max - 2024-01-08 at 09 01 23
Simulator Screen Shot - iPhone 14 Pro Max - 2024-01-08 at 09 01 54
Simulator Screen Shot - iPhone 14 Pro Max - 2024-01-08 at 09 16 48

</details>

<details>
<summary>Android</summary>



</details>

**Comment by shawnborton** [comment]
What exactly are we doing for the bank account icon? Is it the same image being used for both dark and light mode, or something else? 

I wonder if we should follow what we recently did for the fallback avatars where they are different based on the theme. @dubielzyk-expensify does that sound familiar, and if so, can you share what you had for those here?

**Comment by shubham1206agra** [comment]
> What exactly are we doing for the bank account icon? Is it the same image being used for both dark and light mode, or something else?
> 
> I wonder if we should follow what we recently did for the fallback avatars where they are different based on the theme. @dubielzyk-expensify does that sound familiar, and if so, can you share what you had for those here?

They are same before expo too, right?

**Comment by Pujan92** [comment]
> What exactly are we doing for the bank account icon? Is it the same image being used for both dark and light mode, or something else?
 
 Yes, it is the same only. We haven't done anything here for the bank icon color, we just fix the color applied to entire icon for the native.

**Comment by dannymcclain** [comment]
What we have going on here is probably fine (same icon for both themes)—I mean it gets the job done for sure, but I am curious what it would look like with theme-specific colors and if that would be an easy-to-implement thing.

**Comment by shubham1206agra** [comment]
> What we have going on here is probably fine (same icon for both themes)—I mean it gets the job done for sure, but I am curious what it would look like with theme-specific colors and if that would be an easy-to-implement thing.

This will require to create svg using `react-native-svg`, and not as a svg file.

**Review by mountiny — APPROVED** [review]
Thanks for testing @Pujan92 and for the PR @shubham1206agra!

File: src/components/AvatarCropModal/ImageCropView.js [hunk 2]
```diff
@@ -90,7 +90,7 @@ function ImageCropView(props) {
                 <View style={[containerStyle, styles.l0, styles.b0, styles.pAbsolute]}>
                     <Icon
                         src={props.maskImage}
-                        fill={theme.icon}
+                        fill={theme.iconReversed}
```
Comment (situchan) [inline src/components/AvatarCropModal/ImageCropView.js:93]: any reason for keeping this value remained here only? I see all other `fill` values are removed

Comment (Pujan92) [reply src/components/AvatarCropModal/ImageCropView.js:93]: For applying maskImage based on theme we have kept that, comment [ref](https://github.com/Expensify/App/pull/33607#issuecomment-1877418386)

**Pull request merged**